eduroam.md

```diff
@@ -40,98 +53,190 @@ The process of connecting to the network may be different depending on the Linux
-new eduroam setup without needing to run any scripts!
+Your computer should now be connected to the `eduroam` network.
+
+## CLI Setup
+
+Note: Both the `iwd` and `wpa_supplicant` configurations were tested on an otherwise-blank Arch install ISO. `nmcli` configuration was tested on Tails. File location may differ across distributions. If directories don't exist, check your distribution's documentation for the correct location of the files.
+
+### nmcli Configuration
+
+`nmcli` is the terminal interface for NetworkManager. This means that it can either use the `p12` or `pem` variant of your certificate and key. The instructions below are using the `p12` variant. If you wish to use the `pem` variant, replace all instances of the `p12` file with your `pem` file. This information is also useful for troubleshooting the GUI setup, should it work incorrectly.
+
+1. To find out what the name of your network interface is, run `nmcli`. This will return several results. For the examples that follow, this tutorial assumes that `wlan0` is the correct network interface.
+
+2. Open a terminal, and run the following command: 
+```
+nmcli connection add type wifi ifname wlan0 con-name eduroam ssid eduroam
+cd /etc/NetworkManager/system-connections/
+```
+This will add a new connection into NetworkManager's location on the drive, then move you into the directory where it was added. 
+
+3. Open the `eduroam.nmconnection` file in the editor of your choice *as root*. If you do not open it as root, the changes will not be saved.
 
-### wpa_supplicant (CLI) Setup
+4. Edit the file so that it contains the following lines
+```
+[connection]
+id=eduroam
+uuid=
+type=wifi
+interface-name=wlan0
+permissions=
+
+[wifi]
+mac-address-blacklist=
+mode=infrastructure
+ssid=eduroam
+
+[wifi-security]
+auth-alg=open
+key-mgmt=wpa-eap
+
+[802-1x]
+ca-cert=location/of/ritCACert
+client-cert=location/of/p12PrivateKey
+domain-suffix-match=radius.rit.edu
+eap=tls;
+identity=abc1234@rit.edu
+private-key=location/of/p12PrivateKey
+private-key-password=P@ssw0rd1
+
+[ipv4]
+dns-search=
+method=auto
+
+[ipv6]
+addr-gen-mode=stable-privacy
+dns-search=
+method=auto
+```
+This file will contain most of these lines already. The `uuid` field will already be filled in, do not edit this field.
 
-Not all systems are built for GUIs, or have NetworkManager installed. This method, while more involved, should work properly for these types of systems.
+5. Run the following command: 
+```
+# nmcli conection up eduroam
+```
+6. (Optional) Test your connection by running: 
+```
+$ ping 9.9.9.9
+```
+---
+The following configurations must use the listed file types. Use of alternative file types has not been tested, and is not recommended.
+
+### iwd Configuration
 
 If you do not have a `.pem` file from RIT before starting this section, go back to the beginning of this page, and create a new certificate, downloading the `PEM` option instead.
 
```

Comment:
it might be good to either add this disclaimer to each of the individual CLI setup items as the first instruction, or put it at the top of the CLI section (or in the CLI "you will need" section) to begin with

Reply:
Each section has its own information at the beginning on what file it needs. I attempted to get everything to use `p12`, to make life easier, but unfortunately `iwd` doesn't seem to be playing nicely. The tutorial itself in the "Initial Setup" section includes instructions to default to `p12`, but to download the `pem` if you're using `iwd`. Especially if we're planning on separating out the CLI config page (see farther comments), I feel like leaving these notes where they are will be important. 
I'll add a list to the requirements section. If you could check it over once I've done so, to make sure wordings are alright.